tests/llama_index/test_llama_index_tracer.py

```diff
@@ -126,7 +126,7 @@ def test_trace_llm_chat(is_async):
         "raw": ANY,
         "delta": None,
         "logprobs": None,
-        "additional_kwargs": {},
+        "additional_kwargs": {"completion_tokens": 12, "prompt_tokens": 9, "total_tokens": 21},
```

Comment:
I guess we need conditional version logic on this one? Maybe only test in the latest version released since this seems like a newly introduced very useful feature that they added :)